rust/core-lib/src/rpc.rs

```diff
@@ -288,7 +288,11 @@ pub struct EditCommand<T> {
 #[derive(Serialize, Deserialize, PartialEq, Eq, Debug)]
 #[serde(rename_all = "snake_case")]
 pub enum GestureType {
+    Click,
     ToggleSel,
+    RangeSelect,
+    LineSelect,
+    WordSelect,
```

Comment:
Noticing that there's a bit of inconsistency with your names (..Selection) and @raphlinus's ToggleSel. I prefer your names, but at some point when we do a breaking update we might want to standardize these. Fine for now though.